control-plane/api-gateway/binding/result.go

```diff
@@ -183,9 +185,12 @@ func (b bindResults) Condition() metav1.Condition {
 			case errRefNotPermitted:
 				// or if we have a ref not permitted, then use that
 				reason = "RefNotPermitted"
-			case errRouteNoMatchingParent:
-				// or if the route declares a parent that we can't find
-				reason = "NoMatchingParent"
+			case common.ErrExternalRefNotFound:
+				reason = "FilterNotFound"
+				message = result.message
+			case common.ErrInvalidExternalRefType:
+				reason = "UnsupportedValue"
+				message = result.message
```

Comment:
Mostly because common needs to throw the error and the binding package calls common and can't import it. The other option would be to pull these error types into their own package but that feels excessive.

Reply:
No longer handeing errors this way